lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -388,3 +389,25 @@ def save(self, filename, delimiter=";", sampling_width=None):
             with the image. The value indicates the number of pixels in either direction to sum over.
         """
         export_kymolinegroup_to_csv(filename, self._src, delimiter, sampling_width)
+
+    def fit_binding_times(self, n_components):
+        """Fit the distribution of bound dwelltimes to an exponential (mixture) model.
+
+        Parameters
+        ----------
+        n_components : int
+            number of components in the model. Currently only values of {1, 2} are supported.
+        """
+
+        if n_components not in (1, 2):
+            raise ValueError("Only 1- and 2-component exponential distributions are currently supported.")
+
+        image = self[0]._image
+        dwelltimes_sec = np.hstack([line.seconds[-1] - line.seconds[0] for line in self])
+
+        min_observation_time = np.min(dwelltimes_sec)
```

Comment:
This is still true, but the caveat (which took me a while to realize) is that the minimum possible time for kymotracking isn't controlled by the scan time, but rather the parameters set for the tracking algorithm itself. I added a note to the documentation to make this point explicit

Reply:
I agree, the tracking algorithm (combined with the linetime) sets the minimum possible time. Because the tracking algorithm will never mark a single pixel as a line. Good to make this explicit.